### PR TITLE
Docker images

### PR DESCRIPTION
**Pull request by miiu96**
#### Description of the pull request (what is new / what has changed)
- Documented how to use the new docker images for mainnet, devnet and testnet.

#### Did you test the changes locally ?
- [x] yes
- [ ] no

#### Which category(categories) does this pull request belong to?
- [x] document new feature
- [ ] update documentation that is not relevant anymore
- [ ] add examples or more information about a component
- [ ] fix grammar issues
- [ ] other

### CODE REVIEW

File: docs/validators/devnet/use-docker.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+As an alternative to the recommended installation flow, one could choose to run an Elrond Node using the official Docker image: [elrondnetwork/elrond-go-node-devnet](https://hub.docker.com/r/elrondnetwork/elrond-go-node-devnet).
+
+Pull the latest image on an Elrond node with devnet config from Dockerhub
+```
+docker pull elrondnetwork/elrond-go-node-devnet:latest
+```
+
+The following commands runs a Node using the `latest` Docker image and maps a container folder to a local one that holds the necessary configuration:
```
Comment (bogdan-rosianu) [inline docs/validators/devnet/use-docker.md:13]: run* map*

File: docs/validators/devnet/use-docker.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+```
+PATH_TO_BLS_KEY_FILE=/absolute/path/to/bls-key
+
+docker run --mount type=bind,source=${PATH_TO_BLS_KEY_FILE}/,destination=/data elrondnetwork/elrond-go-node-devnet:latest \
+ --validator-key-pem-file="/data/validatorKey.pem"
+```
+
+In the snippet above, make sure you adjust the path to a valid key folder and also provide the appropriate command line arguments to the Node. For more details go to [Node CLI](/validators/node-cli).
```
Comment (bogdan-rosianu) [inline docs/validators/devnet/use-docker.md:22]: valid key file*

File: docs/validators/mainnet/use-docker.md [hunk 3]
```diff
@@ -3,24 +3,23 @@ id: use-docker
 title: How to use the Docker Image
 ---
 
-As an alternative to the recommended installation flow, one could choose to run an Elrond Node using the official Docker image: [elrondnetwork/elrond-go-node](https://hub.docker.com/r/elrondnetwork/elrond-go-node).
-
-The following command runs a Node using the `latest` Docker image and maps a container folder to a local one that holds the necessary configuration:
+As an alternative to the recommended installation flow, one could choose to run an Elrond Node using the official Docker image: [elrondnetwork/elrond-go-node-mainnet](https://hub.docker.com/r/elrondnetwork/elrond-go-node-mainnet).
 
+Pull the latest image on an Elrond node with mainnet config from Dockerhub
```
Comment (bogdan-rosianu) [inline docs/validators/mainnet/use-docker.md:8]: of an

File: docs/validators/mainnet/use-docker.md [hunk 4]
```diff
@@ -3,24 +3,23 @@ id: use-docker
...
 ```
-docker run -d -v /absolute/path/to/config/:/data/ elrondnetwork/elrond-go-node:latest \
- --nodes-setup-file="/data/nodesSetup.json" \
- --p2p-config="/data/config/p2p.toml" \
- --validator-key-pem-file="/data/keys/validatorKey.pem"
+docker pull elrondnetwork/elrond-go-node-mainnet:latest
 ```
 
-In the snippet above, make sure you adjust the path to a valid configuration folder and also provide the appropriate command line arguments to the Node. For more details go to [Node CLI](/validators/node-cli).
+The following commands runs a Node using the `latest` Docker image and maps a container folder to a local one that holds the necessary configuration:
```
Comment (bogdan-rosianu) [inline docs/validators/mainnet/use-docker.md:13]: run* map*

File: docs/validators/testnet/use-docker.md [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+---
+
+As an alternative to the recommended installation flow, one could choose to run an Elrond Node using the official Docker image: [elrondnetwork/elrond-go-node-testnet](https://hub.docker.com/r/elrondnetwork/elrond-go-node-testnet).
+
+Pull the latest image on an Elrond node with testnet config from Dockerhub
+```
+docker pull elrondnetwork/elrond-go-node-testnet:latest
+```
+
+The following commands runs a Node using the `latest` Docker image and maps a container folder to a local one that holds the necessary configuration:
```
Comment (bogdan-rosianu) [inline docs/validators/testnet/use-docker.md:13]: run* map*

File: docs/validators/mainnet/use-docker.md [hunk 6]
```diff
@@ -3,24 +3,23 @@ id: use-docker
...
 
-**Testnet** Validators may switch between different versions of the Node using the following Docker tags:
+docker run --mount type=bind,source=${PATH_TO_BLS_KEY_FILE}/,destination=/data elrondnetwork/elrond-go-node-mainnet:latest \
+ --validator-key-pem-file="/data/validatorKey.pem"
+```
 
-- `elrondnetwork/elrond-go-node:latest`
-- `elrondnetwork/elrond-go-node:devlatest`
-- `elrondnetwork/elrond-go-node:<tag>`, in general form
+In the snippet above, make sure you adjust the path to a valid key folder and also provide the appropriate command line arguments to the Node. For more details go to [Node CLI](/validators/node-cli).
```
Comment (bogdan-rosianu) [inline docs/validators/mainnet/use-docker.md:22]: valid key file*

File: docs/validators/testnet/use-docker.md [hunk 7]
```diff
@@ -0,0 +1,26 @@
...
+
+```
+PATH_TO_BLS_KEY_FILE=/absolute/path/to/bls-key
+
+docker run --mount type=bind,source=${PATH_TO_BLS_KEY_FILE}/,destination=/data elrondnetwork/elrond-go-node-testnet:latest \
+ --validator-key-pem-file="/data/validatorKey.pem"
+```
+
+In the snippet above, make sure you adjust the path to a valid key folder and also provide the appropriate command line arguments to the Node. For more details go to [Node CLI](/validators/node-cli).
```
Comment (bogdan-rosianu) [inline docs/validators/testnet/use-docker.md:22]: valid key file*

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: docs/validators/devnet/use-docker.md (hunk 1 above)
Comment (miiu96) [reply docs/validators/devnet/use-docker.md:13]: done.

File: docs/validators/devnet/use-docker.md (hunk 2 above)
Comment (miiu96) [reply docs/validators/devnet/use-docker.md:22]: done.

File: docs/validators/mainnet/use-docker.md (hunk 3 above)
Comment (miiu96) [reply docs/validators/mainnet/use-docker.md:8]: done.

File: docs/validators/mainnet/use-docker.md (hunk 4 above)
Comment (miiu96) [reply docs/validators/mainnet/use-docker.md:13]: done.

File: docs/validators/testnet/use-docker.md (hunk 5 above)
Comment (miiu96) [reply docs/validators/testnet/use-docker.md:13]: done.

File: docs/validators/mainnet/use-docker.md (hunk 6 above)
Comment (miiu96) [reply docs/validators/mainnet/use-docker.md:22]: done.

File: docs/validators/testnet/use-docker.md (hunk 7 above)
Comment (miiu96) [reply docs/validators/testnet/use-docker.md:22]: done.

**Review by bogdan-rosianu — DISMISSED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**